src/utils/helpers.js

```diff
@@ -253,5 +253,5 @@ export function stringIsIntTheKitchen(str, key, array) {
  * @returns boolean
  */
 export function isInProduction() {
-  return process.env.NODE_ENV === 'production';
+  return process.env.REACT_APP_ENV === 'production';
```

Comment:
Ça change quoi d'utiliser la variable REACT_APP_ENV ?

Reply:
`and when you run npm run build to make a production bundle, NODE_ENV is always equal to 'production'` from https://create-react-app.dev/docs/adding-custom-environment-variables/. Donc quand on buildait pour staging, **NODE_ENV** valait aussi 'production'. C'est la raison pôur laquelle j'ai créé une var env dédiée.